src/components/ArticleCollection.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+import Articles from "../modules/Articles";
+
+import ArticleCard from "./ArticleCard";
+
+const ArticleCollection = () => {
+  const [articles, setArticles] = useState([]);
+
+  const fetchData = async () => {
+    const data = await Articles.index();
+    setArticles(data);
+  };
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const collectionOfArticles = articles.map((article) => {
+    return <ArticleCard article={article} />;
```

Comment:
Each "ArticleCard" must be a `<Card className={classes.card}>` element, this is what you're duplicating.
You don't duplicate the whole container: 
`<div>
      <Container className={classes.cardGrid}>
        <Grid container spacing={4}>
          <Grid item key={article.id} md={12} data-cy="article-collection"> (...)`
Also the "key" attribute is missing.

If you're having troubles let me know we can have a session